src/shogun2-core/src/main/java/de/terrestris/shogun2/model/layer/Layer.java

```diff
@@ -49,17 +65,28 @@ public Layer() {
 
 	/**
 	 * @param name Layer name
-	 * @param type Layer type
-	 * @param source The data source of the layer
-	 * @param appearance The appearance configuration of the layer
 	 */
-	public Layer(String name, String type, LayerDataSource source, LayerAppearance appearance) {
-		super(name, type);
-		this.source = source;
-		this.appearance = appearance;
```

Comment:
I think i just missed that during the refactoring process. I added this constructor again...